GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -32,19 +32,26 @@ public class PlayInstantLoadingScreenGenerator
         private const string LoadingScreenJsonFileName = "LoadingScreenConfig.json";
 
         private static readonly string LoadingScreenScenePath =
-            Path.Combine("Assets", "PlayInstantLoadingScreenAssets");
+            Path.Combine("Assets", "PlayInstantLoadingScreen");
 
-        private static readonly string LoadingScreenJsonPath = Path.Combine(LoadingScreenScenePath, "Resources");
+        private static readonly string LoadingScreenResourcesPath = Path.Combine(LoadingScreenScenePath, "Resources");
 
-        public static string LoadingScreenImagePath;
+        private static string _loadingScreenImagePath;
 
         /// <summary>
         /// Sets the loadingScreenImagePath target for future generated loading screens.
         /// </summary>
-        public static void SetLoadingScreenImagePath()
+        public static void SetLoadingScreenImagePath(string loadingScreenImagePath)
         {
-            LoadingScreenImagePath =
-                EditorUtility.OpenFilePanel("Select Image", "", "png,jpg,jpeg,tif,tiff,gif,bmp");
+            _loadingScreenImagePath = loadingScreenImagePath;
+        }
+        
+        /// <summary>
+        /// Returns the loadingScreenImagePath string
```

Comment:
Nit: in documentation don't list the name of a private field, spell it out in words and add useful info about what it is, e.g. "The path to a fullscreen image displayed in the background while the game loads." (once it's a property).

Reply:
Resolved.